modules/4337/test/eip4337/EIP4337WebAuthn.spec.ts

```diff
@@ -127,7 +127,7 @@ describe('Safe4337Module - WebAuthn Owner', () => {
           module.interface.encodeFunctionData('executeUserOp', [user.address, ethers.parseEther('0.5'), '0x', 0]),
         ]),
         callGasLimit: ethers.toBeHex(2500000),
-        verificationGasLimit: ethers.toBeHex(500000),
+        verificationGasLimit: ethers.toBeHex(580000),
```

Comment:
I would just round this up to `600000` and call it a day. Note that `npm run coverage` sometimes causes issues with the gas limits (as the extra instrumentation instructions add gas overhead).

Reply:
Done 👍🏾